src/sentry/migrations/0126_backfill_platformexternalissue_project_id.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-11-08 04:14
+from __future__ import unicode_literals
+
+from django.db import migrations
+import django.db.models.deletion
+import sentry.db.models.fields.foreignkey
+from sentry.models.platformexternalissue import PlatformExternalIssue
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_platformexternalissue_project_id(apps, schema_editor):
+    """
+    Fill the PlatformExternalIssue.project_id from related Group.project_id.
+    """
+    PlatformExternalIssue = apps.get_model("sentry", "PlatformExternalIssue")
+    all_external_issues = PlatformExternalIssue.objects.filter(project_id__isnull=True).select_related("group")
+    for external_issue in RangeQuerySetWrapperWithProgressBar(queryset=all_external_issues, step=1000):
+        PlatformExternalIssue.objects.filter(id=external_issue.id).update(
```

Comment:
@saifelse Any reason you wouldn't just update `external_issue` and save it? I would think adding the `filter` step here would generate an extra query, yea?

Reply:
This is the most efficient way to do it. It translates to `UPDATE foo SET a=b WHERE x=y`.